doc/install/aspect-setup_stampede.sh

```diff
@@ -4,8 +4,6 @@
 # at 2015 ASPECT Hackathon. Refer to stampede.pdf for
 # further information.
 
-set -x
-
```

Comment:
I like the `set -x`, personally. It makes the script a bit less of a black box, and helps debug if something goes wrong.